docs/en/setup/service-agent/java-agent/Supported-list.md

```diff
@@ -116,6 +116,7 @@ metrics based on the tracing data.
   * JRE Callable and Runnable (Optional²)
 * Cache
   * [Ehcache](https://www.ehcache.org/) 2.x
+  * [GuavaCache](https://github.com/google/guava) 18.x -> 23.x (Optional²)
```

Comment:
In commonly, we have to list all versions, we supported, in `support-version.list`

Reply:
Could we user range to represent multiversion situation. I notice some plugins do not list all versions.

Reply:
We pick some typical releases, such as one version per first-level version release.